frontend/src/metabase/embedding/components/ArchiveButton.module.css

```diff
@@ -0,0 +1,8 @@
+.archiveMenuItem {
+  color: var(--mb-color-danger);
+
+  &:hover:not([data-css-specificity-hack]) {
+    color: var(--mb-color-danger);
```

Comment:
FYI, I think this mb-color-danger color is not part of the theming options but it's fine for now.